artemis-server/src/main/java/org/apache/activemq/artemis/core/server/cluster/impl/BridgeImpl.java

```diff
@@ -570,12 +573,17 @@ public HandleStatus handle(final MessageReference ref) throws Exception {
       if (RefCountMessage.isRefTraceEnabled() && ref.getMessage() instanceof RefCountMessage) {
          RefCountMessage.deferredDebug(ref.getMessage(), "Going through the bridge");
       }
+
+      Exception exception = null;
+      HandleStatus status = null;
```

Comment:
This status value could be final which eliminates the gymnastics done later to capture the result